src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -93,14 +94,15 @@ import {
   SavedObjectsManagementColumnServiceStart,
   SavedObjectsManagementNamespaceServiceStart,
 } from '../../services';
-import { Header, Table, Flyout, Relationships } from './components';
-import { DataPublicPluginStart } from '../../../../../plugins/data/public';
+import { Header, Table, Flyout, Relationships, SavedObjectsDuplicateModal } from './components';
+import { DataPublicPluginStart } from '../../../../data/public';
+
+import { DuplicateMode } from './';
```

Comment:
Should import from `./components` instead of `./index`. Import from `./index` makes the dependency chain confusing.